WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -19,6 +19,12 @@ class PasswordViewController: LoginViewController {
 
     private let isMagicLinkShownAsSecondaryAction: Bool = WordPressAuthenticator.shared.configuration.isWPComMagicLinkShownAsSecondaryActionOnPasswordScreen
 
+    private let configurations = WordPressAuthenticator.shared.configuration
```

Comment:
nit: maybe `configuration` is more consistent with the struct name and other usages?
```suggestion
    private let configuration = WordPressAuthenticator.shared.configuration
```

Reply:
Updated in b014e42e11cd53d1e7ec06c888047db95d8c1f24 👍